static/src/stylesheets/amp/_social.scss

```diff
@@ -31,6 +31,51 @@
     width: 88%;
 }
 
+.social__item--more {
+    float: none;
+    display: inline-block;
+    height: 32px;
+    width: 32px;
+}
+
+.social__item--revealer {
+    display: none;
+
+    &:checked {
```

Comment:
I would maybe just seperate out `.social__item-revealer {}` and  `.social__item-revealer:checked` here as it reduced the complexity of that quite complex block!